apf/apfMesh.cc

```diff
@@ -51,12 +51,22 @@ int const quad_edge_verts[4][2] =
 */
 
 int const tet_edge_verts[6][2] =
+#ifndef HAVE_CAPSTONE
 {{0,1}
 ,{1,2}
 ,{2,0}
 ,{0,3}
 ,{1,3}
 ,{2,3}};
+#else
```

Comment:
Thank you for testing.  I have seen the `parallel_meshgen_*` on the rhel9 systems (https://github.com/SCOREC/core/issues/398); as you said, they are unrelated to this PR.  Ideally, we can quickly resolve the capstone test failures but I won't be opposed to merging this and creating another PR to fix the errors if time is tight/limited.

Reply:
I'm not sure why Morteza added this change (in commit 5def445). I need to make Capstone test meshes with tets because right now all new capstone tests pass when this section is reverted. Reverting just this section fixes bezierElevation, bezierSubdivision, nedelec, and h1_shape_*. However, it doesn't fix bezierMesh or bezierValidity which is beguiling. Is there capstone documentation that would back up the tet-reorder?

Reply:
Ok, so I've checked out the capstone mesh topologies and these are the correct values. Not sure if this will create incompatibilities.